### ISSUE TITLE
on b1b5dfac8e5220254dcb35929fdff212a9ec47eb with typo3 7.6.17-dev jquery backend error

### ISSUE THREAD

**Issue by githubrsys** [TYPO3-Solr/ext-solr#1181]
# Summary
when opening the solr backend and starting an index job the following errors occour

## errors
```
index.php?M=tools_SolrAdministration&moduleToken=e3a4381…&tx_solr_t…:36 Uncaught ReferenceError: jQuery is not defined
    at index.php?M=tools_SolrAdministration&moduleToken=e3a4381…&tx_solr_t…:36
(anonymous) @ index.php?M=tools_SolrAdministration&moduleToken=e3a4381…&tx_solr_t…:36
index.php?M=tools_SolrAdministration&moduleToken=e3a4381…&tx_solr_t…:45 Uncaught ReferenceError: Ext is not defined
    at index.php?M=tools_SolrAdministration&moduleToken=e3a4381…&tx_solr_t…:45
(anonymous) @ index.php?M=tools_SolrAdministration&moduleToken=e3a4381…&tx_solr_t…:45
VM3932 merged-ff7524a…-a9a8cfc….js?1489660821:222 Uncaught ReferenceError: Ext is not defined
    at VM3932 merged-ff7524a…-a9a8cfc….js?1489660821:222
(anonymous) @ VM3932 merged-ff7524a…-a9a8cfc….js?1489660821:222
index.php?M=tools_SolrAdministration&moduleToken=e3a4381…&tx_solr_t…:55 Uncaught TypeError: require is not a function
    at index.php?M=tools_SolrAdministration&moduleToken=e3a4381…&tx_solr_t…:55
(anonymous) @ index.php?M=tools_SolrAdministration&moduleToken=e3a4381…&tx_solr_t…:55
index.php?M=tools_SolrAdministration&moduleToken=e3a4381…&tx_solr_t…:488 Uncaught TypeError: Cannot read property 'setNoTitleString' of undefined
    at index.php?M=tools_SolrAdministration&moduleToken=e3a4381…&tx_solr_t…:488
```

#### solr
b1b5dfac8e5220254dcb35929fdff212a9ec47eb

#### typo3
89355abcd90d2d59291d90e4757da6261f3b9533

**Comment by dkd-kaehm** [comment]
Reproduced with latest 7.6 and ext-solr on dev-master.

But!:
With CMS on dev-master and ext-solr on dev-master all works as expected.

**Comment by timohund** [comment]
@dkd-kaehm this can be closed with the merge of #1300 right?

**Comment by timohund** [comment]
Resolved with #1300
